x/gov/types/expected_keepers.go

```diff
@@ -36,3 +37,8 @@ type StakingKeeper interface {
 	IterateDelegations(ctx sdk.Context, delegator sdk.AccAddress,
 		fn func(index int64, delegation stakingexported.DelegationI) (stop bool))
 }
+
+// AccountKeeper defines the expected account keeper (noalias)
```

Comment:
Maybe we should add these types directly where they're needed (i.e. in `*/simulation/operation`. We don't want to confuse ppl on modules have unnecessary dependencies.

Reply:
we already define all the expected keeper interfaces in one place as a standard so we don't need to define them on the simulation. There's no harm in adding them here imho.